src/main/java/org/akhq/controllers/TopicController.java

```diff
@@ -34,6 +34,7 @@
 import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
 import javax.inject.Inject;
+import org.akhq.models.Record;
```

Comment:
I have compilation problems here, with ambiguous usage of Record (java.lang.Record and Record of package models). I import it explicitly.